blocks/hero/hero.js

```diff
@@ -0,0 +1,28 @@
+export default function decorate(block) {
```

Comment:
Can you make sure that the hero block is always in a new section (either in code or in authoring)? This prevents the padding to be messed up for the following blocks.

Reply:
added to word file